query-node/schemas/councilEvents.graphql

```diff
@@ -0,0 +1,576 @@
+################### Council ####################################################
+
+type AnnouncingPeriodStartedEvent @entity {
```

Comment:
@ondratra 

Just found out during working on the Forum events. We need events to implement the `Event` interface so we could query them at once.

See here: [https://github.com/Joystream/joystream/blob/olympia_dev/query-node/schemas/membershipEvents.graphql#L1](https://github.com/Joystream/joystream/blob/olympia_dev/query-node/schemas/membershipEvents.graphql#L1)